package.json

```diff
@@ -8,7 +8,7 @@
   "license": "Apache-2.0",
   "scripts": {
     "build": "lerna run build && yarn build:schema-api",
-    "build:schema-api": "lerna run extract-api && lerna run generate-plugin-schema",
+    "build:schema-api": "lerna run --ignore @sphereon/ssi-sdk-did-auth-siop-authenticator extract-api && lerna run --ignore @sphereon/ssi-sdk-did-auth-siop-authenticator generate-plugin-schema",
```

Comment:
Have you tested what happens if we remove the build:schema-api step from the respective packages/<name>/package.json? Or to have a command that basically does nothing there? I think that to be a better place to define that no schema generation should be done, than in the parent workspace package.json

Reply:
yep that also works, changed